src/Sulu/Component/Content/Block/BlockProperty.php

```diff
@@ -205,6 +204,13 @@ public function setValue($value)
      */
     public function getValue()
     {
+        // if size of children smaller than minimum
```

Comment:
i dont know why but it doesnt work work without that lines ... it forces when you change the template that there will be empty blocks and the preview found the html of it ...